### DISCUSSION TITLE
[求助]我服了爸爸。。。

### DISCUSSION THREAD

**Discussion by nninng** [share121/inter-knot#2974] [其他]
![7C9BD40B8CD20528917D26C36F688CEF_0](https://github.com/user-attachments/assets/e73a1c57-6198-4fbb-bdce-8652668091fa)

怎么最近玩第五人格遇到的全是伪人？单排人真的太难了，跪求一个免费的双排队友。

**Comment by LShanShi** [comment]
这个委托我接了!

**Comment by weimaojun** [comment]
你好
